pkg/api/pod/warnings_test.go

```diff
@@ -1520,6 +1521,196 @@ func TestWarnings(t *testing.T) {
 				`spec.containers[1].ports[1]: duplicate port definition with spec.containers[1].ports[0]`,
 			},
 		},
+		{
+			name: "create duplicate container ports name in two containers",
+			template: &api.PodTemplateSpec{Spec: api.PodSpec{
+				Containers: []api.Container{
+					{
+						Name: "foo1",
+						Ports: []api.ContainerPort{
+							{ContainerPort: 80, HostPort: 80, Protocol: api.ProtocolUDP, Name: "test"},
```

Comment:
Super duper not, and if this is all I flag I will ignore it: HosttPort is not needed for this test.  I don't think ContainerPort is either, though that is more strictly correct.

Reply:
Removed HostPort from tests. LGTM needed again, thanks @thockin